### ISSUE TITLE
Cannot install on Python 3.9.1 due to version dependency on watchdog

### ISSUE THREAD

**Issue by e8johan** [NiklasRosenstein/pydoc-markdown#172]
I tried installing pydoc-markdown on Debian testing, so Python v3.9.1, and cannot due to a dependency on watchdog<1.0.0, which only works for Python < 3.6.

```
$ pip install pydoc-markdown
Collecting pydoc-markdown
  Using cached pydoc_markdown-3.8.0-py3-none-any.whl (63 kB)
Collecting docspec<0.3.0,>=0.2.0
  Using cached docspec-0.2.0-py3-none-any.whl (6.2 kB)
Collecting nr.collections<0.1.0,>=0.0.1
  Using cached nr.collections-0.0.1.tar.gz (7.8 kB)
Collecting nr.interface<0.1.0,>=0.0.3
  Using cached nr.interface-0.0.4-py2.py3-none-any.whl (11 kB)
Requirement already satisfied: six<2.0.0,>=1.11.0 in ./venv/lib/python3.9/site-packages (from pydoc-markdown) (1.15.0)
Collecting watchdog<1.0.0,>=0.10.2
  Using cached watchdog-0.10.5.tar.gz (99 kB)
ERROR: Package 'watchdog' requires a different Python: 3.9.1 not in '>=2.7, <3.6'
```

```
$ python --version
Python 3.9.1
```

There is a newer version of watchdog, so I guess what is needed is an update to a newer version of watchdog. Both packages claim to work with Python 3.6+, and the API changes for watchdog looks small. Hopefully that means that this is more a matter of updating the pypi metadata, than too much code work.

**Comment by NiklasRosenstein** [comment]
Hey @e8johan , thanks for reporting this. I'm surprised because I had no issue installing Watchdog on Python 3.6. I'll take a closer look this weekend.

**Comment by elizabeth-bruce-getupside** [comment]
Hey folks - I happened to also encounter this issue very recently on Python 3.6.

I took a quick look into `watchdog`'s release history - it looks like recent versions up to `v0.10.4` supported Python version 2.7 and 3.4+, whereas the last two pre-1.0 releases, `v0.10.5` and `v0.10.6`, only support Python versions 2.7 and 3.4-3.5. Since the dependency for `watchdog` here is `>=0.10.2,<1.0.0`, it looks like this repo started pulling in one of the last two pre-1.0 releases when they were released on December 10th. This also explains how this issue manifested so suddenly.

I concur with @e8johan - hopefully this should be a relatively easy fix by bumping the dep to post-1.0?
